lvm_read.py

```diff
@@ -42,6 +42,21 @@ def _lvm_dump(lvm_data, filename, protocol=-1):
     pickle.dump(lvm_data, output, protocol=protocol)
     output.close()
 
+def get_separator(file):
+    separators = {'Tab': '\t',
+                  'Comma': ','
+    }
+    for line in file:
+        if line.startswith('Separator'):
+            separator = line.strip()[10:]
+            break
+    file.seek(0) 
+    if separator in separators.keys():
+        return separators[separator]
+    else:
+        raise Warning('No separator defined, using tabulator!')
```

Comment:
The specification says that the separator header is optional. A warning should not be triggered from absence.